README.md

```
@@ -1,21 +1,33 @@
-1) Prereqs (only need to run once):
+Azure DevOps (formerly VSTS) extension that integrates [TestingBot](https://testingbot.com)
+into your pipelines: a configuration task that exports your TestingBot credentials
+and optionally launches TestingBot Tunnel, a task to stop the tunnel, and a build
+results tab.
+
+## Building the plugin
+
+Requires **Node.js >= 20**.
+
+1) Install dependencies (once):
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)